pkg/metrics/bootstrap.go

```diff
@@ -0,0 +1,62 @@
+package metrics
+
+import (
+	"context"
+
+	"github.com/spf13/viper"
+
+	prometheusclient "github.com/prometheus/client_golang/prometheus"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/metric/prometheus"
+	"go.opentelemetry.io/otel/metric/global"
+	export "go.opentelemetry.io/otel/sdk/export/metric"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/histogram"
+	controller "go.opentelemetry.io/otel/sdk/metric/controller/basic"
+	processor "go.opentelemetry.io/otel/sdk/metric/processor/basic"
+	selector "go.opentelemetry.io/otel/sdk/metric/selector/simple"
+	"go.opentelemetry.io/otel/sdk/resource"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+
+	v1 "github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
+	"github.com/jaegertracing/jaeger-operator/pkg/tracing"
+)
+
+const meterName = "jaegertracing.io/jaeger"
+
+// Bootstrap configures the OpenTelemetry meter provides with the prometheus exporter
+func Bootstrap(ctx context.Context, namespace string, client client.Client) error {
+	if viper.GetBool("operand-metrics-enabled") {
```

Comment:
If you invert the check, you can just return `nil` at the beginning of the function, letting the remaining of the func to reside outside the conditional, simplifying the understanding of this func in the long term.